tests/c/udp-push-pop.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
```

Comment:
Fixed comment style for the header.

I agree on changing c89 to c99 comments (which introduced one-liners), but I disagree talking about using C++ standards in a C file.

Reply:
Fixed comment style for the header.

I agree on changing c89 to c99 comments (which introduced one-liners), but I disagree talking about using C++ standards in a C file.

I really dislike mixing languages :D

Reply:
Okay, on not mixing languages.  But I'm glad C added C++ comments to c99, so we can use them :-)

I think we left this as you were going to look info if the auto-documentation tags work with c99 comments before potentially switching the comment style to all c99.

Reply:
Did you determine there was a problem with going to c99 style comments?

Reply:
Well, Doxygen kind of supports C99 comments: [Doxygen Manual: Documenting the code](https://www.doxygen.nl/manual/docblocks.html).

It would require a triple ‘///’ comment style to identify something as a documentation block.

Note however that, other .c and .h files don’t rely on this ‘///’ comment style for documentation blocks.

Could I fix the comment style for other files later, on a different PR?

Reply:
Okay, I'll go ahead and approve.